decidim-core/config/initializers/devise.rb

```diff
@@ -301,9 +309,9 @@
   # If you want to use other strategies, that are not supported by Devise, or
   # change the failure app, you can configure them inside the config.warden block.
   #
-  # config.warden do |manager|
-  #   manager.failure_app = Decidim::DeviseFailureApp
-  # end
+  config.warden do |manager|
+    manager.failure_app = RandomStalling
```

Comment:
Shouldn't the strategy be the same as  in https://github.com/decidim/decidim/pull/5553/files#diff-763c8f6d1cbc9555804240779718829c ?

Reply:
@mrcasals there's this question pending